src/chainparams.cpp

```diff
@@ -215,6 +215,10 @@ class CMainParams : public CChainParams {
         // Avian BIP44 cointype in mainnet is '175'
         nExtCoinType = 175;
 
+        vector<FounderRewardStructure> rewardStructures = {  {INT_MAX, 5}// 5% founder/dev fee forever
+                                        										   };
+        consensus.nFounderPayment = FounderPayment(rewardStructures, 250);
```

Comment:
"250" needs to be updated to the latest block height

Reply:
Agreed as we will be activating down the line to avoid problems with existing chain work